src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/page/Entity/Pages/ExtraFunctions.php

```diff
@@ -45,5 +43,5 @@ public function getPageTemplates()
      */
     public function getDefaultView()
     {
-        return '{{ bundle }}:Pages:Common/view.html.twig';
+        return '{% if not isV4 %}{{ bundle.getName() }}/{%endif%}Pages/Common/view.html.twig';
```

Comment:
```suggestion
        return '{% if not isV4 %}{{ bundle.getName() }}:{%endif%}Pages/Common/view.html.twig';
```

There should be a ":" after the bundlename, not a /